joss_paper.bib

```diff
@@ -65,21 +64,24 @@ @article{slaughter2021continuous
 }
 
 @article{lindsay2021automatic,
-  title={Automatic Differentiation in MetaPhysicL and Its Applications in MOOSE},
-  author={Lindsay, Alexander and Stogner, Roy and Gaston, Derek and Schwen, Daniel and Matthews,
+  title={Automatic Differentiation in {MetaPhysicL} and Its Applications in {MOOSE}},
+  author={Lindsay, Alexander D. and Stogner, Roy and Gaston, Derek and Schwen, Daniel and Matthews,
   Christopher and Jiang, Wen and Aagesen, Larry K and Carlsen, Robert and Kong, Fande and Slaughter,
-  Andrew and others},
+  Andrew and Permann, Cody and Martineau, Richard},
   journal={Nuclear Technology},
-  pages={1--18},
+  pages={905--922},
+  volume={207},
+  number={7},
   year={2021},
-  publisher={Taylor \& Francis}
+  publisher={Taylor \& Francis},
+  doi={10.1080/00295450.2020.1838877}
 }
 
 @techreport{sqa,
-  title={Software Quality Assurance Plan for MOOSE and MOOSE-Based Applications (PLN-4005)},
+  title={Software Quality Assurance Plan for {MOOSE} and {MOOSE}-Based Applications ({PLN}-4005)},
   institution={Idaho National Laboratory},
   year={2020},
-  note = {Revision 7}
+  note = {Revision 7},
```

Comment:
I don't think a comma should go after the last item, but maybe this is a moot point if this reference needs to be removed anyway (I'm currently asking around if there's a public SQA plan reference).

Reply:
This is just force of habit on my part—commas after the final list entry are totally acceptable, but yes unnecessary. (Same is true in most programming language lists, dicts, etc.).